roles/swift-account/tasks/main.yml

```diff
@@ -13,6 +13,14 @@
             dest=/etc/swift/account-server.conf owner=swift group=swift
   notify: restart swift-account service
 
+- name: trigger restart on upgrades
```

Comment:
Okay, please do. We'll want the handler that's currently being used to restart services on config changes (e.g. https://github.com/blueboxgroup/ursula/blob/master/roles/swift-container/tasks/main.yml#L15 ) is restarting the correct services. That's the one we'll use for package changes too.